dm/tests/dmctl_command/run.sh

```diff
@@ -467,6 +467,12 @@ function run_validator_cmd {
 		"new\/ignored\/resolved: 4\/0\/0" 1 \
 		"\"stage\": \"Running\"" 4 \
 		"\"stage\": \"Stopped\"" 1
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Comment:
add a case when validator is not enabled

Reply:
[1565730](https://github.com/pingcap/tiflow/pull/5181/commits/1565730b6e4d7f9a64cbbbea5153bc8ca8ad3078)